### ISSUE TITLE
No manual interaction = ban

### ISSUE THREAD

**Issue by BitScorpio** [rs-peer/issues#404]
I've been testing things around the recent bans occuring upon logout/worldhop/shutdown on newly created accounts.

I botted everything from account creation, going through tutorial island, botting some method and muling.

Accounts that had me clicking somewhere (I just tested changing tabs using my mouse in the tutorial island) did not get banned.

Accounts that had 0 manual interaction got banned within ~10 seconds after switching worlds in-game using the WorldHopper.hopTo method or using Game.getClient().setWorld then relogging.

I tried using the deprecated Mouse.click method to emulate an actual mouse click and bans still occurred.

To sum it up, accounts that were interfered with using mouse clicks (Keyboard not tested) were not banned while other accounts that only had interactions done via the RSPeer API were banned.

**Comment by Lezor91** [comment]
Another user said the same thing but said using Mouse#click stopped the bans, this didn't make sense to me either as RSPeer already uses that. I tested this multiple times with a few other scripters (followed steps mentioned by you in the discord) and we did not get banned. If this was something that I could reproduce easily myself then I'd look into it, but since that's not the case I'm going to have to assume it's something very specific that you're doing. I'm going to close this as it is misleading to others

**Comment by tim-vu** [comment]
> Another user said the same thing but said using Mouse#click stopped the bans, this didn't make sense to me either as RSPeer already uses that. I tested this multiple times with a few other scripters (followed steps mentioned by you in the discord) and we did not get banned. If this was something that I could reproduce easily myself then I'd look into it, but since that's not the case I'm going to have to assume it's something very specific that you're doing. I'm going to close this as it is misleading to others

I can reproduce this every single time on a fresh account without primes. See the video I posted in the scripters channel.
